addons/l10n_id_efaktur/models/account_move.py

```diff
@@ -129,7 +137,10 @@ def download_efaktur(self):
                 raise ValidationError(_('Could not download E-faktur in draft state'))
 
             if record.partner_id.l10n_id_pkp and not record.l10n_id_tax_number:
-                raise ValidationError(_('Connect %(move_number)s with E-faktur to download this report', move_number=record.name))
+                if not self.l10n_id_need_kode_transaksi:
+                    raise ValidationError(_('E-faktur is not available for invoices without any taxes.'))
```

Comment:
pot file I think ?

Reply:
Wouldn't it be nice to add, even if in another PR?
Not a ID expert, but I'm not sure if English is known enough over there for all users?